doc/source/security/data-masking.rst

```diff
@@ -0,0 +1,58 @@
+.. _security/data-masking:
+
+==================================================================
+Data Masking
+==================================================================
```

Comment:
Should be documented already - is this going to be a part of 8.0.17?

Reply:
As far as I know, yes. Data masking is part of 17.

Reply:
ah, thanks. I have confirmed it with Zsolt. It is not yet merged, but will be soon.